File: polyglot/windows/scripting/seis_windows_platform.lua
local capability = {
  id = "windows-lua-scripting",
  languages = { "Lua", "PowerShell", "Rust", "SQL" },
  quality_gates = {
    "lua_syntax_when_available",
    "windows_path_safety",
    "permission_scope",
    "offline_fallback",
    "event_log_awareness"
  },
  offline_helper = true,
  remote_bridge = true
}

return capability

File: polyglot/windows/scripting/seis_windows_platform.rb
module SEIS
  module WindowsPlatform
    LANGUAGES = ["Ruby", "PowerShell", "Go", "SQL"].freeze
    QUALITY_GATES = [
      "ruby_syntax_when_available",
      "windows_path_safety",
      "permission_scope",
      "offline_fallback",
      "event_log_awareness"
    ].freeze

    def self.ready_for_seis_agent?
      LANGUAGES.length >= 4 && QUALITY_GATES.length >= 5
    end
  end
end
